src/DeviceInput/InputDevices/deviceSourceManager.ts

```diff
@@ -133,6 +147,24 @@ export class DeviceSourceManager implements IDisposable {
         return this._devices[deviceType].filter((source) => { return !!source; });
     }
 
+    /**
+     * Returns a read-only list of all available devices
+     * @returns Read-only array with active devices
+     */
+    public getListofDevices(): ReadonlyArray<{ deviceType: DeviceType, deviceSlot: number }> {

```

Comment:
The `o` in `of` is not capitalized? Maybe just `getDevices`?

Reply:
I can change this to getDevices